crime-applications-adaptor/application/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/PassportedMapperTest.java

```diff
@@ -89,4 +93,17 @@ void shouldMap_Income_Support_BenefitType() {
     Passported actualPassported = passportedMapper.map(crimeApplyMaatApplicationExternal);
     assertTrue(actualPassported.getBenefitIncomeSupport());
   }
+
+  @Test
+  void shouldMap_Partner_BenefitType() {
+    MaatApplicationExternal crimeApplyMaatApplicationExternal = TestData.getMaatApplication();
+    Partner partner = crimeApplyMaatApplicationExternal.getClientDetails().getPartner();
+    partner.setBenefitType(Partner.BenefitType.JSA);
+    partner.setLastJsaAppointmentDate(LocalDate.parse("2020-12-23"));
+    Passported actualPassported = passportedMapper.map(crimeApplyMaatApplicationExternal);
```

Comment:
Minor point, I would break up this test method with some white space to emphasise the "given, when, then" structure.

Reply:
done